panel/src/components/Dialogs/PageCreateDialog.vue

```diff
@@ -3,6 +3,7 @@
     ref="dialog"
     :button="$t('page.draft.create')"
     :notification="notification"
+    :data-redirect="redirect"
```

Comment:
What is this line for?

Reply:
That's a great question. I put it to get the `redirect` parameter from `PagesSection.vue`, but I think it has nothing to do with 😄 

If there is no big problem, I would be glad if you can make the last touch 🙏